utils/cleanupResources.ts

```diff
@@ -8,23 +11,27 @@ for (const envVar of ['PINECONE_API_KEY']) {
     console.log(`INFO Found environment variable ${envVar} in .env file`);
   }
 }
-
-var pinecone = require('../dist');
-
 (async () => {
   const p = new pinecone.Pinecone();
 
   const collectionList = await p.listCollections();
-  for (const collection of collectionList.collections) {
-    console.log(`Deleting collection ${collection.name}`);
-    await p.deleteCollection(collection.name);
+  if (collectionList.collections) {
```

Comment:
Same here -- this was throwing errors in my IDE because `collectionList.collections` could be undefined.